sycl/include/sycl/multi_ptr.hpp

```diff
@@ -487,7 +480,7 @@ class __SYCL_TYPE(multi_ptr) multi_ptr<const void, Space, DecorateAddress> {
   multi_ptr(accessor<ElementType, Dimensions, Mode, target::device,
                      isPlaceholder, PropertyListT>
                 Accessor)
-      : multi_ptr(detail::cast_AS<decorated_type *>(
+      : multi_ptr(detail::static_address_cast<Space>(
```

Comment:
`target::device` accessor does point to the `global_space` (or `ext_intel_global_device_space` when the extension is used).